capstone/src/main/webapp/index.html

```diff
@@ -4,18 +4,98 @@
+      <div class="row">
+        <div class="col s12 m12 l12">
+          <ul class="tabs">
+            <li class="tab col s4"><a href="#terms_of_usage">Terms of Usage</a></li>
+            <li class="tab col s4"><a href="#about" >About</a></li>
+            <li class="tab col s4"><a href="#get_started">Get Started</a></li>
+          </ul>
+          <div class="col s12 tb" id="terms_of_usage">
+            <h3 class="white-text text-darken-4">Terms Of Usage</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.
+            </p>
+          </div>
+           <div class="col s12 tb" id="about">
+            <h3 class="white-text text-darken-4">About</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.
+            </p>
+          </div>
+           <div class="col s12 tb" id="get_started">
+            <h3 class=" white-text text-darken-4">Get Started</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.</p>
+          </div>
+        </div>   
+      </div>   
+    </section>  
+
+    <footer class="page-footer">
+       <div class"container">
+           © 2020 Studentia inc
+       </div>
+    </footer>
+
+    <!--JavaScript at end of body for optimized loading-->
+    <script type="text/javascript" src="js/materialize.min.js"></script>
+    <script src="script.js"></script>
+    <script type = "text/javascript" src = "https://code.jquery.com/jquery-2.1.1.min.js"></script>             
+    <script src = "https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/js/materialize.min.js"></script> 
+
+    <script>
```

Comment:
Move this block into script.js: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns